internal/security/secretstore/secure-messagebus.go

```diff
@@ -40,7 +40,7 @@ default:
   port: 6379
   protocol: redis
   server: localhost
-  serviceServer: http://localhost:48080
+  connectionSelector: edgex.redisMsgBus
```

Comment:
We need to make sure `serviceServer` is not used by eKuiper 1.3.1, because the docker compose scripts are still set to use that version ([example](https://github.com/edgexfoundry/edgex-compose/blob/9455f9fb605d96c929c856f304b3d95d29e81b74/docker-compose.yml#L729)).

Reply:
Thanks. "Since eKuiper v1.2.1, we will only support EdgeX v2(Ireland and later)....EdgeX source does not rely on Core contract Service anymore. Users can remove the serviceServer property in the edgex.yaml configuration." ([link](https://github.com/lf-edge/ekuiper/blob/master/docs/en_US/edgex/edgex_rule_engine_tutorial.md#migrate-to-edgex-v2))